Documentation/CodeSnippets/Manual/Entity/FrontendTypoScript.rst.txt

```diff
@@ -33,18 +33,36 @@
 
       :returntype: `array`
 
+   .. php:method:: hasSetup()
```

Comment:
Please add a "versionadded:: 13.1" (or something) to the according page where this is used, so people know when this was added.

Reply:
This is only an automatic update of the Code-Snippets. There will be a changelog entry about that. Such changes are out of scope here.

Reply:
Nope, they are not. You change content, so you should take care that they are complete and consistent. Otherwise they may get lost.
I the past, I created one PR for that API change with the corresponding change in the rst. But it's up to you. Won't approve that, as information is missing.